bfillings/vsearch.py

```diff
@@ -222,6 +222,43 @@ def getHelp(self):
         return help_str
 
 
+def parse_uc_to_clusters(uc_filepath, seed_id_field=8):
+    """ Parse VSEARCH .uc file to return clusters
+
+    Parameters
+    ----------
+    uc_filepath: string
+        path to .uc file
+    seed_id_field: integer
+        .uc field to use as seed ID
+
+    Returns
+    -------
+    clusters: dictionary
+        dictionary of clusters
+    """
+    clusters = {}
+    with open(uc_filepath, 'U') as uc_f:
+        for line in uc_f:
+            line = line.split()
+            if line[0] == 'S':
+                seed_id = line[seed_id_field]
+                if seed_id not in clusters:
+                    clusters[seed_id] = [seed_id]
+                else:
+                    raise ValueError(
+                        "Multiple definition of the same seed: %s" % seed_id)
+            if line[0] == 'H':
```

Comment:
should it also include an `else` in case that is not and 'H' or an 'S' and raise an error?